src/frontend/mosh-server.cc

```diff
@@ -185,8 +189,11 @@ int main( int argc, char *argv[] )
        && (strcmp( argv[ 1 ], "new" ) == 0) ) {
     /* new option syntax */
     int opt;
-    while ( (opt = getopt( argc - 1, argv + 1, "i:p:c:svl:" )) != -1 ) {
+    while ( (opt = getopt( argc - 1, argv + 1, "i:p:c:svl:A" )) != -1 ) {
```

Comment:
It'd be nice if we didn't need another option here, and could learn this by sending the option in the packets.  But the server starts the child shell immediately, and needs to know whether or not to set SSH_AUTH_SOCK.  Perhaps the server could wait until receiving packets before starting the child shell?

(Definitely a bikeshed.)